tests/test_authority.py

```diff
@@ -78,6 +78,13 @@ def test_invalid_host_skipping_validation_can_be_turned_off(self):
         except:  # Could be requests...RequestException, json...JSONDecodeError, etc.
             pass  # Those are expected for this unittest case
 
+    def test_ciam_authority_should_be_path_less(self):
+        http_client = MinimalHttpClient()
+        with self.assertRaises(ValueError):
+            Authority('https://contoso.ciamlogin.com/undesirable_path', http_client)
+        Authority('https://msidlabciam1.ciamlogin.com', http_client)
```

Comment:
@jmprieur et al, this test case rejects a ciam authority with path.

Reply:
It's actually fine to let it go through (custom domains)

Reply:
Thanks. I am convinced that MSAL should not accept path-less only. This behavior and test case have been removed in latest push.